bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -305,20 +308,45 @@ impl LockTime {
         }
     }
 
+    /// Returns true if this [`absolute::LockTime`] is satisfied by `other` lock.
+    ///
+    /// This function is useful if you wish to check a lock time against various `other` locks e.g.,
+    /// filtering out locks which cannot be satisfied.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use bitcoin::absolute::{LockTime, LockTime::*};
+    /// let lock_time = LockTime::from_consensus(100);
+    /// let check = LockTime::from_consensus(100 + 1);
+    /// let is_satisfied = lock_time.is_satisfied_by_lock(check);
+    /// assert!(is_satisfied);
+    /// ```
+    pub fn is_satisfied_by_lock(&self, other: LockTime) -> bool {
+        use LockTime::*;
+
+        match (*self, other) {
+            (Blocks(n), Blocks(m)) => n <= m,
+            (Seconds(n), Seconds(m)) => n <= m,
```

Comment:
Please, use more meaningful names for `n` and `m`. I suggest `this` and `other`

Reply:
Sure thing, will fix. Thanks